test/account_tree.test.ts

```diff
@@ -4,7 +4,7 @@ import { Tree, Hasher } from "./utils/tree";
 
 let DEPTH: number;
 let BATCH_DEPTH: number;
-contract("Account Tree", accounts => {
+describe("Account Tree", async () => {
```

Comment:
If you don't use `Contract.deployed()` but use `Contract.new()`, use `describe`. It doesn't run truffle-fixture.js